packages/quantic/cypress/e2e/default-2/search-box/search-box.cypress.ts

```diff
@@ -0,0 +1,634 @@
+
+            scope('when focusing on the search box input', () => {
+              Actions.focusSearchBox(textarea);
+              cy.wait(InterceptAliases.QuerySuggestions);
+
+              Expect.displaySuggestionList(true);
+              Expect.displayClearRecentQueriesButton(true);
+              Expect.numberOfQuerySuggestions(
+                expectedDisplayedSuggestions.length
+              );
+              Expect.querySuggestionsEquals(expectedDisplayedSuggestions);
+            });
+          });
+        });
+
+        describe('when clearing the recent queries', () => {
+          const exampleRecentQueries = ['foo', 'bar'];
+
+          beforeEach(() => {
+            setRecentQueriesInLocalStorage(exampleRecentQueries);
+            visitSearchBox({...defaultOptions, textarea});
+            mockQuerySuggestions(exampleQuerySuggestions);
+          });
+
+          it('should clear the recent queries and send the right analytics', () => {
+            const expectedQuerySuggestions = [
+              ...exampleRecentQueries,
+              ...exampleQuerySuggestions,
+            ];
+            scope('when loading standalone search box', () => {
+              Expect.displaySearchBoxInput(true, textarea);
+              Expect.displaySearchButton(true);
+            });
+
+            scope('when focusing on the search box input', () => {
+              Actions.focusSearchBox(textarea);
+              cy.wait(InterceptAliases.QuerySuggestions);
+
+              Expect.displaySuggestionList(true);
+              Expect.displayClearRecentQueriesButton(true);
+              Expect.numberOfQuerySuggestions(expectedQuerySuggestions.length);
+              Expect.querySuggestionsEquals(expectedQuerySuggestions);
+            });
+
+            scope('when clicking on the clear recent queries button', () => {
+              Actions.clickClearRecentQueriesButton();
+
+              Expect.logClearRecentQueries();
+              Expect.displayClearRecentQueriesButton(false);
+              Expect.displaySuggestionList(false);
+            });
+
+            scope('when selecting a query suggestion', () => {
+              Actions.focusSearchBox(textarea);
+              cy.wait(InterceptAliases.QuerySuggestions);
+              Expect.displaySuggestionList(true);
+              Expect.numberOfQuerySuggestions(exampleQuerySuggestions.length);
+              Expect.querySuggestionsEquals(exampleQuerySuggestions);
+              const clickedSuggestionIndex = 0;
+              Actions.clickQuerySuggestion(clickedSuggestionIndex);
+              Expect.searchWithQuery(
+                exampleQuerySuggestions[clickedSuggestionIndex],
+                {
+                  LSkey: recentQueriesLSKey,
+                  queries: [exampleQuerySuggestions[clickedSuggestionIndex]],
+                }
+              );
+            });
+          });
+        });
+
+        describe('when using the keyboard to select a recent query', () => {
```

Comment:
Not a huge deal, but personally I would've built these tests this way.
You don't have to change it.

```
describe('when using the keyboard', () => {
  it('can select recent queries', () => {});
  it('can select query suggestions', () => {});
});
```